### ISSUE TITLE
Add pid_aux option to [Debug]

### ISSUE THREAD

**Issue by loganharbour** [idaholab/moose#16154]
## Reason
Many times I find myself adding the following:
```
[AuxVariables/pid]
  family = MONOMIAL
  order = CONSTANT
[]

[AuxKernels/pid]
  type = ProcessorIDAux
  variable = pid
[]
```
when debugging in parallel. 

It would be nice to have the option to do
```
[Debug]
  pid_aux = true
[]
```
or
```
<executable> <options> Debug/pid_aux=true
```

## Design
Add `pid_aux` to the `SetupDebugAction`, which adds the objects in the input shown above.

## Impact
New capability.
